index.js

```diff
@@ -1,25 +1,245 @@
-const navBar = document.querySelector('#nav-bar');
-const menuBtn = document.querySelector('#menu-btn');
-const exitBtn = document.querySelector('#exit-btn');
-const links = document.querySelectorAll('.nav-item');
-    
-
-
-const menuActivity = () => {
-    const visibility = navBar.getAttribute('data-visible');
-    if(visibility === 'false'){
-        navBar.setAttribute('data-visible', true)
-    }else if(visibility === 'true'){
-        navBar.setAttribute('data-visible', false)
-    }
+ "use strict";
+ const cardArray = [ 
+  {
+    name: 'Tonic',
+    feature: 'CANOPY',
+    background: '• Back-end Dev •',
+    year: '2015',
+    image: './images/Portfolio.jpg',
+    large: './images/Portfolio3x.png',
+    description:'Experimental content creation feature that allows users to add to an existing story over the course of a day without spamming their friends.',
+    message:' Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry standard dummy text ever since the 1500s.',
+    html: 'html',
+    css: 'css',
+    javascript: 'javascript',
+    ruby: 'ruby on rails',
+    bootstrap: 'bootstrap',
+    github: 'github',
+    linkLive: 'https://github.com/Munyabelden/munyabelden.github.io/',
+    linkSource: 'https://github.com/Munyabelden/munyabelden.github.io',
+  },
+  {
```

Comment:
- [ ] [**OPTIONAL**] Kindly improve the position of these links :+1: , it will make your page sticks to the Figma design :+1: 
![Screenshot from 2023-02-09 21-52-08](https://user-images.githubusercontent.com/80458199/217923160-54a94244-694b-4b6e-989d-d557a5ee480c.png)